src/main/java/org/jlibvips/VipsAngle.java

```diff
@@ -26,7 +26,7 @@ public static VipsAngle fromInteger(int value) {
             case 180: return D180;
             case 270: return D270;
             default:
-                throw new IllegalArgumentException("Allowed VipsAngle's are [0°, 90°, 180°, 270°].");
```

Comment:
There was some ASCII encoding error for these characters on my mac, so I removed them